draft-ietf-quic-qlog-h3-events.md

```diff
@@ -530,21 +560,17 @@ headers: [
 ~~~
 {: #h3-headersframe-ex title="H3HeadersFrame example"}
 
-~~~ cddl
-H3HeadersFrame = {
-    frame_type: "headers"
-    headers: [* H3HTTPField]
-}
-~~~
-{: #h3-headersframe-def title="H3HeadersFrame definition"}
-
-~~~ cddl
-H3HTTPField = {
-    name: text
-    ? value: text
-}
-~~~
-{: #h3field-def title="H3HTTPField definition"}
+{{Section 4.2 of RFC9114}} and {{Section 5.1 of RFC9110}} define rules for the
+characters used in HTTP field sections names and values. Characters outside the
+range are invalid and result in the message being treated as malformed.
+
```

Comment:
```suggestion
range are invalid and result in the message being treated as malformed. It can however be useful to also log these invalid HTTP fields.
```